packages/expo-screen-orientation/ios/ScreenOrientationRegistry.swift

```diff
@@ -0,0 +1,239 @@
+import Foundation
+import ExpoModulesCore
+protocol OrientationListener {
+  func screenOrientationDidChange(_ orientation: UIInterfaceOrientation)
+}
+
+// This singleton holds information about desired orientation for every app which uses expo-screen-orientation.
+// Marked @objc and public, because this it is also used in EXAppViewController.
```

Comment:
```suggestion
/**
 The singleton that holds information about desired orientation for every app which uses expo-screen-orientation.
 Marked @objc and public, because this it is also used in EXAppViewController.
 */
```